style.css

```diff
@@ -512,3 +512,109 @@ button:active {
   border-radius: 8px;
   color: #36b37e;
 }
+
+.contact-form {
+  background-color: #fff;
+  max-width: var(--sect-max-width);
+  margin: 4% auto 0 auto;
+  display: flex;
+  flex-flow: row wrap;
+  align-items: flex-start;
+  justify-content: center;
+  gap: 24px;
+  background-image: url("Visual/contact-right-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 17%;
+  background-position: bottom right;
+}
+
+.contact-form h2 {
+  margin-bottom: 5px;
+}
+
+.contact-container {
+  width: 90%;
+  max-width: 684px;
+}
+
+.contact-container ul {
+  margin: auto;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas: "full-name" "email" "message";
+  gap: 24px;
+  list-style: none;
+}
+
+.full-name {
+  grid-area: full-name;
+}
+
+.email {
+  grid-area: email;
+}
+
+.message {
+  grid-area: message;
+}
+
+button[type="submit"] {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  padding: 0 12px;
+  cursor: pointer;
+  align-items: center;
```

Comment:
Im really impress with your code!!! It is really amazing!

I sugest to delete align-itemes, beause it is repeating twice in your code and probably show an error.

Reply:
Thank you!!